backend/node/genvm/base.py

```diff
@@ -258,24 +279,51 @@ async def run_contract(
                 else:
                     function_to_run(*args)
             except Exception as e:
-                print("\nError running contract", e)
-                print(f"\n{traceback.format_exc()}")
+                trace = traceback.format_exc()
                 error = e
+                print("Error executing method", error)
+                print(trace)
```

Comment:
if we have the logger below, I don't think we should have prints. I think we should go only with the logger

Reply:
This is a little confusing I admit, but basically those get buffered here and a little down they get printed directly to terminal indeed. The reasoning is to avoid fancy formatting for better readability of multiine tracebacks in this instance, and in parallel also sentd to the frontend of course.